expo/expo-cli
# Updated webpack version

## How

- bump expo-cli, xdl, and webpack-config webpack deps
- bump other deps in webpack-config
- add react-refresh plugin
- fix SW register by passing force to copy webpack plugin

## Test Plan

- Built and published a website
  - Lighthouse scores are still at 100
- Ensure homepage works
- Ensure SW is registered properly
- Ensure PWA assets can be generated
- Test fast refresh with `EXPO_WEBPACK_FAST_REFRESH=1` in a blank project and in native-component-list.
- Test HTTPS works 
- [x] Test Next.js projects continue to work
